PurrSoft/PurrSoft.Application/Queries/FosterQueries/FosterQueries.cs

```diff
@@ -0,0 +1,30 @@
+﻿using PurrSoft.Application.Common;
+using PurrSoft.Application.Models;
+using PurrSoft.Application.QueryOverviews;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PurrSoft.Application.Queries.FosterQueries
+{
+	public class GetFilteredFostersQueries : BaseRequest<CollectionResponse<FosterOverview>>
+	{
+		public int Skip { get; set; }
+		public int Take { get; set; }
+		public string? SortBy { get; set; }
+		public string? SortOrder { get; set; }
+		public string? Search { get; set; }
+		public string? Status { get; set; }
+		public string? Location { get; set; }
+		public string? ExperienceLevel { get; set; }
+		public bool? HasOtherAnimals { get; set; }
+		public int? AnimalFosteredCount { get; set; }
+	}
+
+	public class GetFosterByIdQuery : BaseRequest<FosterOverview>
```

Comment:
So overviews are used for lists.
Like it is considered smaller than a dto.
So if you return by id here you should return a FosterDto.